app/views/applications/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>New Application</h1>
+<%= form_with url: "/applications", method: :post, local: true do |f| %>
```

Comment:
Make sure to use data: {turbo: false} instead of local:true in the future, just in case, to avoid Turbo issues.